chromadb/server/fastapi/__init__.py

```diff
@@ -744,6 +745,34 @@ async def delete_collection(
             limiter=self._capacity_limiter,
         )
 
+    @staticmethod
+    def _generate_ids_when_not_present(
```

Comment:
I don't think this method can live here? What happens in local chroma?

Reply:
<img width="1295" alt="Screenshot 2024-08-22 at 12 48 33 AM" src="https://github.com/user-attachments/assets/5a00dc91-53d2-499a-a861-571dc2cebc44">

Reply:
Yeah I don't think this change is fully considered, not sure if its halfway but this has to live on layer lower if you want it at the "conceptual server" layer.

Would also like to have testing that would catch this as opposed to me reading it.